GitUI/CommandsDialogs/SettingsDialog/Pages/ConfirmationsSettingsPage.Designer.cs

```diff
@@ -106,9 +108,31 @@ private void InitializeComponent()
             this.tableLayoutPanel3.RowStyles.Add(new System.Windows.Forms.RowStyle());
             this.tableLayoutPanel3.RowStyles.Add(new System.Windows.Forms.RowStyle());
             this.tableLayoutPanel3.RowStyles.Add(new System.Windows.Forms.RowStyle());
-            this.tableLayoutPanel3.Size = new System.Drawing.Size(442, 253);
+            this.tableLayoutPanel3.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.tableLayoutPanel3.Size = new System.Drawing.Size(442, 276);
             this.tableLayoutPanel3.TabIndex = 1;
             // 
+            // chkRebaseConfirmation
+            // 
+            this.chkRebaseConfirmation.AutoSize = true;
+            this.chkRebaseConfirmation.Location = new System.Drawing.Point(3, 256);
+            this.chkRebaseConfirmation.Name = "chkRebaseConfirmation";
+            this.chkRebaseConfirmation.Size = new System.Drawing.Size(397, 17);
+            this.chkRebaseConfirmation.TabIndex = 11;
+            this.chkRebaseConfirmation.Text = "Rebase (Won\'t show the rebase dialog unless rebasing with advance options)";
```

Comment:
I want to invert the setting and change the text to
Always show the rebase dialog

If not inverted, change the name to
Only show the rebase dialog with Advanced options

Reply:
I prefer inverting it, but the group head contradicts.
![image](https://user-images.githubusercontent.com/596334/38455435-a1692b1e-3aaa-11e8-9d3f-c36ac797a0fa.png)

Reply:
With that heading, it is hard to write something that is useful at all. Keep what you have.

Reply:
This setting should not go here, this is the reason you struggle to find a good title for it.
These are advanced setting disabling confirmation (lowering safety). None of them should be checked by default but this new setting is. It just does not belong here.

Reply:
I consider rebase without confirmation as advanced. So if this is not advanced, Eben's previous implementation is a better fit.
My vote has less weight here.

Regardless of where the checkbox is, my remaining request is to get the popup by default. (ThreeState could be used to get a popup too, I updated my comment.)

Reply:
@gerhardol I did not say it is not an advanced setting. It just does not look like a confirmation. It can be placed under the advanced node, but should not be grouped with confirmations. Also any setting categorized as advanced should default to the safe option not requiring advanced knowledge, not the other way around.

Reply:
I am not entirely convinced either that this is the best place for the new setting.
If we insist on adding the checkbox here, I'm finding the proposed wording excessive.
Perhaps we can invert the negative and trim to something like the following:

```
Don't ask to confirm to (use with caution):

* Rebase (the dialog will only be shown if rebasing with advance options)`
```